scaffold/Taskfile.yml

```diff
@@ -4,3 +4,8 @@ dotenv: ['.env']
 
 includes:
   drupal: ./vendor/lullabot/drainpipe/tasks/drupal.yml
+
+tasks:
+  default:
+    cmds:
+      - echo "Hello, world!"
```

Comment:
newline?

I wonder how useful this is, given we could point users to https://taskfile.dev/#/usage. If we keep it, we should add a description so `task -l` doesn't error out.

Reply:
The Taskfile won't validate without at least one command

Reply:
That's unfortunate, given that there are other tasks included. Worth opening an upstream bug? Task will work without a task defined, so I wonder if it's just a schema issue.